packages/environment-check/README.md

```diff
@@ -0,0 +1,41 @@
+# @sap-ux/environment-check
```

Comment:
The `README` needs some information about how to use the CLI that the module exposes.

Reply:
Added